src/v2/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -154,6 +155,7 @@ export const FollowArtistPopoverRowFragmentContainer = createFragmentContainer(
   {
     artist: graphql`
       fragment FollowArtistPopoverRow_artist on Artist {
+        slug
```

Comment:
`non-blocking`: There's also the `href` field on `Artist` which will give you `/artist/${artist.slug}`